packages/hash/api/src/seed-data/index.ts

```diff
@@ -0,0 +1,99 @@
+import { Logger } from "@hashintel/hash-backend-utils/logger";
+import { GraphApi } from "@hashintel/hash-graph-client";
+import { OrgModel, OrgSize } from "../model";
+import { workspaceAccountId } from "../model/util";
+import { getRequiredEnv } from "../util";
+import { ensureDevUsersAreSeeded } from "./dev-users";
+import { PageDefinition, seedPages } from "./seed-pages";
+
+// Seed Org with some pages.
+const seedOrg = async (params: {
+  graphApi: GraphApi;
+  logger: Logger;
+}): Promise<OrgModel> => {
+  const { graphApi, logger } = params;
+
+  const sharedOrgModel = await OrgModel.createOrg(graphApi, {
+    name: getRequiredEnv("WORKSPACE_ACCOUNT_NAME"),
+    shortname: getRequiredEnv("WORKSPACE_ACCOUNT_SHORTNAME"),
```

Comment:
Can we please import these directly from `backend-utils/src/system` instead?

Reply:
Was not aware of those, done in 2e75f3cb5b47a7846eff3a7fe32171121a565274